src/tutorial/index.rst

```diff
@@ -169,27 +169,27 @@ Before the model description, we will also define two (constant) variables, to d
 
   .. code-tab:: cpp C++
 
-    #include "flamegpu/flamegpu.h"
+    ...
+    // all code examples are assumed to be implemented within a main function. E.g. int main(int argc, const char **argv)
```

Comment:
Convention is `const char * argv[]`. Would be good to split over two lines to avoid horizontal scroll bars for narrow windows (e.g. phones).

Reply:
```suggestion
    // All code examples are assumed to be implemented within a main function.
    // E.g. int main(int argc, const char *argv[])
```